containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -389,7 +430,15 @@ void loadServicePrivateKey() {
             sslContextFactory.setWantClientAuth(true);
         }
 
-        sslContextFactory.setEnableOCSP(enableOCSP);
+        if (enableOCSP) {
+            // https://stackoverflow.com/questions/49904935/jetty-9-enable-ocsp-stapling-for-domain-validated-certificate
+            Security.setProperty("ocsp.enable", "true");
```

Comment:
not sure what these three system properties are for. If necessary, please follow up with Jetty / webtide support to see exactly what we need to set to enable ocsp in jetty.

Reply:
Look at the link. Joakim is from Jetty / webtide support.